src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -26,28 +27,35 @@ const InfoViewer = ({
     size,
     className,
     multilineLabels,
-}: InfoViewerProps) => (
-    <div className={b({size}, className)}>
-        {title && <div className={b('title')}>{title}</div>}
-        {info && info.length > 0 ? (
-            <div className={b('items')}>
-                {info.map((data, infoIndex) => (
-                    <div className={b('row')} key={data.label + infoIndex}>
-                        <div className={b('label')}>
-                            <div className={b('label-text', {multiline: multilineLabels})}>
-                                {data.label}
+    renderOnEmptyState,
+}: InfoViewerProps) => {
+    if ((!info || !info.length) && renderOnEmptyState) {
+        return renderOnEmptyState();
```

Comment:
i'd pass displayable data there
surely title
maybe size

Reply:
What do you mean by displayable data here? If you mean `info`, what the sense in passing it here if it is an empty array or undefined?

Reply:
surely title
maybe size